bcipy/helpers/load.py

```diff
@@ -1,56 +1,49 @@
+from datetime import datetime
 from tkinter import Tk
 import numpy as np
 import pandas as pd
 import logging
 from codecs import open as codecsopen
 from json import load as jsonload
+from shutil import copyfile
+from pathlib import Path
 import pickle
+from bcipy.helpers.parameters import Parameters, DEFAULT_PARAMETERS_PATH
 
 from tkinter.filedialog import askopenfilename, askdirectory
 
 log = logging.getLogger(__name__)
 
+DEFAULT_PARAMETERS_PATH = 'bcipy/parameters/parameters.json'
```

Comment:
It looks like we should pull this from from `bcipy.helpers.parameters`?

Reply:
Yes, good point. I'll clean that up.